<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmComputador</class>
 <widget class="QMainWindow" name="FrmComputador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manutenção de Computadores</string>
  </property>
  <property name="toolTip">
   <string>Cadastro de computadores</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="abas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Manutenção de Computadores</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabCadastro">
     <attribute name="icon">
      <iconset>
       <normaloff>:/icones/COPY.BMP</normaloff>:/icones/COPY.BMP</iconset>
     </attribute>
     <attribute name="title">
      <string>Cadastro</string>
     </attribute>
     <attribute name="toolTip">
      <string>Manutenção dos registros de Computadores</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Código do computador</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Descrição:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edDescricao">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Descrição do computador</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edCodComputador">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>10</number>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Código do computador</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkUso">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Em condições de uso?  </string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnConfirmarProcura">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>14</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Procurar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabListagem">
     <attribute name="icon">
      <iconset>
       <normaloff>:/icones/WINPREV.BMP</normaloff>:/icones/WINPREV.BMP</iconset>
     </attribute>
     <attribute name="title">
      <string>Listagem</string>
     </attribute>
     <attribute name="toolTip">
      <string>Lista os registros de Computadores</string>
     </attribute>
     <widget class="QTableWidget" name="grdComputador">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>771</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <column>
       <property name="text">
        <string>Código</string>
       </property>
       <property name="background">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Descrição</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Pode usar?</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>RA com reserva</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Horário reservado</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>174</x>
        <y>0</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);</string>
      </property>
      <property name="title">
       <string>Ordem</string>
      </property>
      <widget class="QRadioButton" name="rbPorCodigo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>por Código</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbPorDescricao">
       <property name="geometry">
        <rect>
         <x>115</x>
         <y>20</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>por Descrição</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbPorCondicao">
       <property name="geometry">
        <rect>
         <x>241</x>
         <y>20</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>por Condição</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCadastro">
    <property name="title">
     <string>Cadastro</string>
    </property>
    <addaction name="actionConex_o"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuListagem">
    <property name="title">
     <string>Listagem</string>
    </property>
    <addaction name="actionListar"/>
   </widget>
   <widget class="QMenu" name="menuSair">
    <property name="title">
     <string>Sair</string>
    </property>
    <addaction name="actionSair_2"/>
   </widget>
   <addaction name="menuCadastro"/>
   <addaction name="menuListagem"/>
   <addaction name="menuSair"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="tbBotoes">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolTip">
    <string>Botões de ação</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Inicio"/>
   <addaction name="action_Anterior"/>
   <addaction name="action_Proximo"/>
   <addaction name="action_Fim"/>
   <addaction name="separator"/>
   <addaction name="action_Buscar"/>
   <addaction name="separator"/>
   <addaction name="action_Novo"/>
   <addaction name="action_Editar"/>
   <addaction name="action_Salvar"/>
   <addaction name="separator"/>
   <addaction name="action_Cancelar"/>
   <addaction name="separator"/>
   <addaction name="action_Excluir"/>
   <addaction name="separator"/>
   <addaction name="action_Sair"/>
  </widget>
  <action name="actionConex_o">
   <property name="text">
    <string>Conexão</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionListar">
   <property name="text">
    <string>Listar</string>
   </property>
  </action>
  <action name="actionSair_2">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="action_Novo">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/botoes/Add.bmp</normaloff>:/icones/botoes/Add.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Novo</string>
   </property>
   <property name="toolTip">
    <string>Incluir novo registro</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Editar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/botoes/COPY.BMP</normaloff>:/icones/botoes/COPY.BMP</iconset>
   </property>
   <property name="text">
    <string>&amp;Editar</string>
   </property>
   <property name="toolTip">
    <string>Altera dados do registro em exibição</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_Salvar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/botoes/Save.bmp</normaloff>:/icones/botoes/Save.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Salvar</string>
   </property>
   <property name="toolTip">
    <string>Grava o registro atual no banco de dados</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Excluir">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/botoes/Minus.bmp</normaloff>:/icones/botoes/Minus.bmp</iconset>
   </property>
   <property name="text">
    <string>E&amp;xcluir</string>
   </property>
   <property name="toolTip">
    <string>Exclui o registro exibido na tela</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Cancelar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/botoes/UNDO.BMP</normaloff>:/icones/botoes/UNDO.BMP</iconset>
   </property>
   <property name="text">
    <string>&amp;Cancelar</string>
   </property>
   <property name="toolTip">
    <string>Cancela a operação atualmente em execução</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_Buscar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/botoes/FIND.BMP</normaloff>:/icones/botoes/FIND.BMP</iconset>
   </property>
   <property name="text">
    <string>&amp;Buscar</string>
   </property>
   <property name="toolTip">
    <string>Busca o registro cujo código foi digitado e o exibe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_Inicio">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/botoes/first.bmp</normaloff>:/icones/botoes/first.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Início</string>
   </property>
   <property name="toolTip">
    <string>Posiciona visualização no primeiro registro e o exibe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_Anterior">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/botoes/prior.bmp</normaloff>:/icones/botoes/prior.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Anterior</string>
   </property>
   <property name="toolTip">
    <string>Posiciona visualização no registro anterior ao atualmente exibido</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Proximo">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/botoes/next.bmp</normaloff>:/icones/botoes/next.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Próximo</string>
   </property>
   <property name="toolTip">
    <string>Posiciona visualização no registro seguinte ao atual</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Fim">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/botoes/last.bmp</normaloff>:/icones/botoes/last.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Fim</string>
   </property>
   <property name="toolTip">
    <string>Posiciona visualização no último registro e o exibe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Sair">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/icones/botoes/CLOSE1.BMP</normaloff>:/icones/botoes/CLOSE1.BMP</iconset>
   </property>
   <property name="text">
    <string>Sai&amp;r</string>
   </property>
   <property name="toolTip">
    <string>Fecha a conexão, salva dados e fecha o formulário</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
